test/data_structure/bitmap/check_bitmap.c

```diff
@@ -41,9 +41,10 @@ END_TEST
 START_TEST(test_bitset_getset)
 {
     int i, j;
+    int cols_size = sizeof(cols)/sizeof(uint16_t);
```

Comment:
prefer you just add a `#define COLS_SIZE 4` above, and declare cols as `static uint16_t cols[COLS_SIZE]`

Reply:
Done.